app/models/btc_payment.rb

```diff
@@ -17,7 +17,7 @@ def self.payments_before_date(date)
 
   def self.create_entry(destination, public_address, float_amount)
     self.create!({
-      btcusdprice_at_moment: OpenAssets::Transaction.new.get_btc_spot_price_coinbase()*100,
+      btcusdprice_at_moment: OpenAssets::Transactions.new.get_btc_spot_price_coinbase()*100,
```

Comment:
Line is too long. [92/80]<br>Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Do not use parentheses for method calls with no arguments.<br>Surrounding space missing for operator '*'.